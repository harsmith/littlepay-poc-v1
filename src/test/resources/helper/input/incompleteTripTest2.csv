ID,DateTimeUTC,TapType,StopId,CompanyId,BusID,PAN
1,27-01-2018 13:00:00,ON,Stop2,Company100,Bus56,4484070000000000