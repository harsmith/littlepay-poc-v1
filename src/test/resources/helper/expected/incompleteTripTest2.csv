Started,Finished,DurationSecs,FromStopId,ToStopId,ChargeAmount,companyId,BusID,PAN,Status
27-01-2018 13:00:00,27-01-2018 13:00:00,0,Stop2,Stop2,$5.50,Company100,Bus56,4484070000000000,INCOMPLETE
